### ISSUE TITLE
"Hide screen toggle" API presenter option not working on Win native 

### ISSUE THREAD

**Issue by LeahW** [Screenleap/support#55]
Impact: Confirmed on API Demo page when using Win native. Only occurs if this presenter option is initially not checked, then checked on subsequent shares without quitting the app in between sessions

Steps to Reproduce:
1. On a PC, go tohttp://www.screenleap.com/api/demo
2. Start a screen share using Native app
3. End screen share
4. Click to Show presenter options and check the box for "Hide screen toggle"
5. Start another Native app share
6. The dropdown to toggle between 'Share inside rectangle' and 'Entire screen' still appears in control panel

Workaround: Quit the Screenshare app after step 3 and it will correctly implement the "Hide screen toggle" option

**Comment by plimul** [comment]
Fixed in itoggle branch: commit 414470722e22bdd1422aadba7ff900f865b16a6b

**Comment by ghost** [comment]
Fixed in itoggle branch: commit https://github.com/Screenleap/screenleap/commit/414470722e22bdd1422aadba7ff900f865b16a6b

**Comment by tuyentruong** [comment]
Merged into develop
